### PR TITLE
Bug 1809197 - Add dltoken to telemetry events and clients_daily

### PR DESCRIPTION
**Pull request by relud**
cc @ncalexan

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/telemetry_derived/event_events_v1/query.sql [hunk 1]
```diff
@@ -27,7 +27,8 @@ WITH base AS (
     environment.settings.attribution.source AS attribution_source,
```
Comment (ncalexan) [inline sql/moz-fx-data-shared-prod/telemetry_derived/event_events_v1/query.sql:27]: Not up here, with its siblings?

File: sql/moz-fx-data-shared-prod/telemetry_derived/main_events_v1/query.sql [hunk 2]
```diff
@@ -29,7 +26,8 @@ SELECT
   environment.settings.attribution.source AS attribution_source,
```
Comment (ncalexan) [inline sql/moz-fx-data-shared-prod/telemetry_derived/main_events_v1/query.sql:26]: Likewise, not up here?

**Review by ncalexan — APPROVED** [review]
Yes, I think these tables make sense.  Thanks for getting to this promptly, @relud!

File: sql/moz-fx-data-shared-prod/telemetry_derived/event_events_v1/query.sql (hunk 1 above)
Comment (relud) [reply sql/moz-fx-data-shared-prod/telemetry_derived/event_events_v1/query.sql:27]: good point, moved

**Review by scholtzan — APPROVED** [review]

**Pull request merged**